daceml/onnx/op_implementations/pure_implementations.py

```diff
@@ -467,3 +487,67 @@ def prog(input, output):
             output[:] = dace.elementwise(lambda x: x, input)
 
         return program_for_node(prog, sdfg, state, node).to_sdfg()
+
+
+@autoregister_params(op="Gemm", name="pure")
+class PureGemm(ONNXForward):
+    @staticmethod
+    def forward_can_be_applied(node: ONNXOp, state: SDFGState,
+                               sdfg: SDFG) -> bool:
+        if node.alpha == 1.0 and node.beta == 1.0 and node.transA == 0 and node.transB == 1:
+            return True
+        return False
+
+    @staticmethod
+    def forward(node: ONNXOp, state: SDFGState,
+                sdfg: SDFG) -> typing.Union[Node, SDFG]:
+        node.validate(sdfg, state)
+
+        assert node.alpha == 1.0 and node.beta == 1.0 and node.transA == 0 and node.transB == 1
+
+        # the gemm libnode is broken for now, so we just do it manually
+        atype = in_desc_with_name(node, state, sdfg, "A")
+        if "C" in node.in_connectors:
+
+            def prog(A, B, C, Y):
+                Y[:] = A @ np.transpose(B) + C
+        else:
+
+            def prog(A, B, Y):
+                Y[:] = A @ np.transpose(B)
```

Comment:
you really don't need this anymore, especially for cases where alpha/beta are not 1,1

Reply:
I will rewrite this implementation in a later PR now that the Gemm libnode supports `C`, if that's fine by you.

Reply:
yes.